README.md

```diff
@@ -13,17 +13,23 @@ To use the latest stable version of the Solidity compiler via Node.js you can in
 npm install solc
 ```
 
-### Using on the command line
+Alternatively, in order to get the latest development version, you can install the version available here on GitHub:
```

Comment:
I was thinking about this and I am not entirely sure I it is a good idea suggesting that.